### PR TITLE
Taskfile download button and improved how to start readme

### PR DESCRIPTION
**Pull request by rick-nu**
# What

- The `README.md` has been updated with more clear instructions how to get started with a `./Taskfile` (when you're not familiar with it)
- A download button has been added as a Quick and easy way to directly insert your new Taskfile into your project
- Removes npm config init task as React 19 has been released

## Screenshot

image

### CODE REVIEW

File: src/components/Generator/GeneredTaskfile/SaveFile/save-file.module.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.copy:hover,
+.download:hover {
+	color: #fff;
+	border-color: #fff;
+}
+
+.copy.copied,
+.copy:hover.copied {
```
Comment (nickspaargaren) [inline src/components/Generator/GeneredTaskfile/SaveFile/save-file.module.scss]:
```suggestion
.copy.copied:hover {
```

File: src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		elem.href = window.URL.createObjectURL(blob);
+		elem.download = 'Taskfile';
+		document.body.appendChild(elem);
+		elem.click();
+		document.body.removeChild(elem);
+	};
+
+	return (
+		<div className={styles.container}>
+			<button type="button" className={`${styles.download}`} onClick={download} title="Download Taskfile" />
```
Comment (nickspaargaren) [inline src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx]:
```suggestion
			<button type="button" className={styles.download} onClick={download} title="Download Taskfile" />
```

File: src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import { ReactElement, useState } from 'react';
+
+import styles from './save-file.module.scss';
+
+type CopyProps = {
+	content: string;
+};
+
+const CopyToClipboard = ({ content }: CopyProps): ReactElement => {
```
Comment (nickspaargaren) [inline src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx]:
```suggestion
const SaveFile = ({ content }: CopyProps): ReactElement => {
```

**Review by nickspaargaren — CHANGES_REQUESTED** [review]
The CSS is solid but could benefit from modern layout techniques and a bit more accessibility focus. These improvements will enhance scalability, maintainability, and user experience.

File: src/components/Generator/GeneredTaskfile/SaveFile/save-file.module.scss (hunk 1 above)
Comment (rick-nu) [reply src/components/Generator/GeneredTaskfile/SaveFile/save-file.module.scss]: Fixed, thank  you for this valuable comment :+1:

File: src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx (hunk 3 above)
Comment (rick-nu) [reply src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx]: Fixed, thank  you for this valuable comment :+1:

File: src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx (hunk 2 above)
Comment (rick-nu) [reply src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx]: Fixed, thank  you for this valuable comment :+1:

**Review by nickspaargaren — APPROVED** [review]

**Pull request merged**